### ISSUE TITLE
Signups are currently disabled status 400 bad request

### ISSUE THREAD

**Issue by gabe1314** [withfig/fig#2701]
### Checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have run `fig restart` and replicated the issue again

### Operating system

macOS 13.4.1 (22F770820d)

### Expected behaviour

I'm not able to sign into Fig with my Github account, I've tried to use doctor and re-install but it's not working. 

### Actual behaviour

❯ fig -help

  ███████╗██╗ ██████╗
  ██╔════╝██║██╔════╝
  █████╗  ██║██║  ███╗
  ██╔══╝  ██║██║   ██║
  ██║     ██║╚██████╔╝
  ╚═╝     ╚═╝ ╚═════╝ CLI

╭────────────────────────────────────────────────────╮
│ fig            Open the Fig Dashboard              │
│ fig doctor     Debug Fig installation issues       │
╰────────────────────────────────────────────────────╯

 Popular Subcommands           Usage: fig [subcommand]
╭────────────────────────────────────────────────────╮
│ run            Execute a Fig Script                │
│ settings       Customize appearance & behavior     │
│ tweet          Tweet about Fig                     │
│ update         Check for updates                   │
│ issue          Create a new GitHub issue           │
│ quit           Quit the Fig app                    │
╰────────────────────────────────────────────────────╯

 To see all subcommands, use:
  > fig help-all

❯ fig doctor

✘ Logged into Fig: Not logged in. Run `fig login` to login.

error: Not logged in. Run `fig login` to login.
❯ fig update
No updates available,
2.16.0 is the latest version.
❯ fig settings
❯ fig run
error: No token
❯ fig
❯ fig restart
Launching Fig...
❯ brew reinsall fig
Error: Unknown command: reinsall
❯ fuck
brew install fig [enter/↑/↓/ctrl+c]
Running `brew update --auto-update`...
==> Auto-updated Homebrew!
Updated 4 taps (homebrew/cask-fonts, mongodb/brew, homebrew/core and homebrew/cask).
==> New Formulae
aerleon             massdriver          tpm                 yazi
legitify            notation            wtfis
==> New Casks
dockx                      font-zhuque-fangsong       json-viewer

You have 27 outdated formulae and 3 outdated casks installed.

Warning: Not upgrading fig, the latest version is already installed
❯ brew reinstall fig
==> Caveats
Please launch the Fig application to finish setup.

==> Downloading https://repo.fig.io/generic/stable/asset/2.16.0/universal/fig.dm
Already downloaded: /Users/gabrielsandoval/Library/Caches/Homebrew/downloads/8cfcc98ca7e0528d885c4e428e9dc7e27600b373bd15168511f7890c9cbb6a09--fig.dmg
==> Uninstalling Cask fig
==> Removing launchctl service io.fig.launcher
Password:
==> Removing launchctl service io.fig.uninstall
==> Removing launchctl service io.fig.dotfiles-daemon
==> Quitting application 'com.mschrage.fig'...
Application 'com.mschrage.fig' quit successfully.
==> Running uninstall script /Applications/Fig.app/Contents/MacOS/fig-darwin-uni
==> Backing App 'Fig.app' up to '/opt/homebrew/Caskroom/fig/2.16.0/Fig.app'
==> Removing App '/Applications/Fig.app'
Warning: Your terminal does not have App Management permissions, so Homebrew will delete and reinstall the app.
This may result in some configurations (like notification settings or location in the Dock/Launchpad) being lost.
To fix this, go to System Settings > Privacy & Security > App Management and add or enable your terminal.
==> Unlinking Binary '/opt/homebrew/bin/fig'
==> Purging files for version 2.16.0 of Cask fig
==> Installing Cask fig
==> Moving App 'Fig.app' to '/Applications/Fig.app'
==> Linking Binary 'fig-darwin-universal' to '/opt/homebrew/bin/fig'
🍺  fig was successfully installed!
❯ fig
Launching Fig...
❯ fig
Launching Fig...
❯ fig logout
You are now logged out
Run fig login to log back in to Fig
❯ fig login

First copy your one-time code: 9916-3CA6
Press Enter to open github.com in your browser...
Opened https://github.com/login/device in your browser

error: Signups are currently disabled (status: 400 Bad Request)
❯
❯
❯
❯ fig doctor

✘ Logged into Fig: Not logged in. Run `fig login` to login.

error: Not logged in. Run `fig login` to login.
❯ fig login

First copy your one-time code: 1491-81D1
Press Enter to open github.com in your browser...
Opened https://github.com/login/device in your browser

error: Signups are currently disabled (status: 400 Bad Request)

### Steps to reproduce

Logout and back in if you're not already logged out. 

### Environment

```yaml
❯ fig diagnostic

fig-details:
  - 2.16.0
hardware-info:
  - model:
  - model-id:
  - chip-id: Apple M2
  - cores: 8
  - mem: 16.00 GB
os-info:
  - macOS 13.4.1 (22F770820d)
environment:
  - shell: /bin/zsh
  - terminal: iterm
  - cwd: /Users/gabrielsandoval
  - exe-path: /opt/homebrew/bin/fig
  - install-method: brew
  - env-vars:
    - SHELL: /bin/zsh
    - __CFBundleIdentifier: com.googlecode.iterm2
    - PATH: /opt/homebrew/bin:/opt/homebrew/sbin:/Library/Frameworks/Python.framework/Versions/3.9/bin:/usr/local/bin:/System/Cryptexes/App/usr/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/share/dotnet:/opt/X11/bin:/Library/Apple/usr/bin:/Library/Frameworks/Mono.framework/Versions/Current/Commands:/Applications/Wireshark.app/Contents/MacOS:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/local/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/bin:/var/run/com.apple.security.cryptexd/codex.system/bootstrap/usr/appleinternal/bin
    - DISPLAY: /private/tmp/com.apple.launchd.wrOubcuuPD/org.macosforge.xquartz:0
    - TERM: xterm-256color
```

**Comment by JackFannon** [comment]
@brendanfalk said this in the Fig discord on 24/08:
> Hey guys - we've made a temporary change that disables new user sign ups as we push out some new functionality. Should have communicated this better. We should have it back online soon. Existing users are not affected. Really really sorry about this!

and then on 25/08 said they would have more updates on Monday (28/08).

**Comment by PhilippDehler** [comment]
Tried it today. It's still not working

**Comment by mave007** [comment]
Still not working:

```fig login

First copy your one-time code: FBA8-32D0
Press Enter to open github.com in your browser...
Opened https://github.com/login/device in your browser

error: Signups are currently disabled (status: 400 Bad Request)```

**Comment by grant0417** [comment]
For more info here you can see this blog post: https://fig.io/blog/post/fig-joins-aws

TL;DR signups are disabled for now while we Fig integrates with AWS
